test/farm/001-farm.bats

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bats
+#
+# Tests of podman farm commands
+#
+
+load helpers.bash
+
+###############################################################################
+# BEGIN tests
+
+fname="test-farm"
+containerfile="test/farm/Containerfile"
+
+@test "farm - check farm has been created" {
+    run_podman farm ls
+    assert "$status" -eq 0 "get list of farms"
+    assert "$output" =~ $fname
+    assert "$output" =~ "test-node"
+}
+
+@test "farm - build on local only" {
+    iname="test-image-1"
+    empty_farm="empty-farm"
+    # create an empty farm
+    run_podman farm create $empty_farm
+    assert "$status" -eq 0 "create empty farm"
+    run_podman farm --farm $empty_farm build -f $containerfile -t $iname .
+    assert "$status" -eq 0 "create new manifest list $iname"
+    assert "$output" =~ "Local builder ready"
+    # inspect manifest list built and saved in dir
+    run_podman manifest inspect $iname
+    assert "$status" -eq 0 "inspect manifest list $iname"
+    assert "$output" =~ "amd64"
```

Comment:
There is an entire Red Hat team dedicated to multiarch testing. This is going to fail when it gets to their hands.

Reply:
Fixed this to grab the system's architecture and compare against it instead.